### PR TITLE
Update debian to fix duplicated tag

### PR DESCRIPTION
**Pull request by tianon**
See also https://github.com/docker-library/official-images/pull/382, which will prevent future occurrances of this problem. :angel:

### CODE REVIEW

File: library/debian [hunk 1]
```diff
@@ -1,33 +1,31 @@
...
-jessie: git://github.com/tianon/docker-brew-debian@a13096487c65c924e476d86547c1578e711d3d2a jessie
+8.0: git://github.com/tianon/docker-brew-debian@9a6aebe153f72ffc623a7f692625c2b69b3ccb50 jessie
+8: git://github.com/tianon/docker-brew-debian@9a6aebe153f72ffc623a7f692625c2b69b3ccb50 jessie
+jessie: git://github.com/tianon/docker-brew-debian@9a6aebe153f72ffc623a7f692625c2b69b3ccb50 jessie
 
-oldstable: git://github.com/tianon/docker-brew-debian@a13096487c65c924e476d86547c1578e711d3d2a oldstable
+oldstable: git://github.com/tianon/docker-brew-debian@9a6aebe153f72ffc623a7f692625c2b69b3ccb50 oldstable
 
-8.0: git://github.com/tianon/docker-brew-debian@a13096487c65c924e476d86547c1578e711d3d2a sid
-8: git://github.com/tianon/docker-brew-debian@a13096487c65c924e476d86547c1578e711d3d2a sid
```
Comment (tianon) [inline library/debian]: Dem LOLs.

Comment (tianon) [inline library/debian]: "sid ate my lunch"

**Comment by tianon** [comment]
Build test of #383; eb6be04de2edbdff9a3845eaa1151f5d91a2433f (`debian`):

```console
$ bashbrew build https://raw.githubusercontent.com/docker-library/official-images/eb6be04de2edbdff9a3845eaa1151f5d91a2433f/library/debian
Fetching debian (git://github.com/tianon/docker-brew-debian) ...
Processing debian:8.0 ...
Processing debian:8 ...
Processing debian:jessie ...
Processing debian:oldstable ...
Processing debian:sid ...
Processing debian:6.0.10 ...
Processing debian:6.0 ...
Processing debian:6 ...
Processing debian:squeeze ...
Processing debian:stable ...
Processing debian:testing ...
Processing debian:unstable ...
Processing debian:7.7 ...
Processing debian:7 ...
Processing debian:wheezy ...
Processing debian:latest ...
Processing debian:rc-buggy ...
Processing debian:experimental ...
```

**Comment by yosifkit** [comment]
LGTM

**Pull request merged**